var/spack/repos/builtin/packages/arrow/package.py

```diff
@@ -15,6 +15,7 @@ class Arrow(CMakePackage, CudaPackage):
     homepage = "https://arrow.apache.org"
     url      = "https://github.com/apache/arrow/archive/apache-arrow-0.9.0.tar.gz"
 
+    version('7.0.0',  sha256='74f1e1f1240edadc55eb27f1b6a1b99cd3ddbc3ef7cca9b357b0e5c9c0225622')
```

Comment:
Where is the sha256 coming from?

When I `spack checksum arrow@7.0.0` I get:

```suggestion
    version('7.0.0',  sha256='57e13c62f27b710e1de54fd30faed612aefa22aa41fa2c0c3bacd204dd18a8f3')
```

Reply:
I don't remember, but you're definitely right. Thanks!